helm/publisher/templates/web/deployment.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-web
+  labels:
+    app: {{ .Release.Name }}-web
+spec:
+  replicas: {{ .Values.webReplicaCount}}
+  selector:
+    matchLabels:
+      app: {{ .Release.Name }}-web
+  template:
+    metadata:
+      labels:
+        app: {{ .Release.Name }}-web
+      annotations:
+        # TODO: if Helm library is used, `$.Template.BasePath` needs to be replaced 
+        # by chart name, see https://helm.sh/docs/howto/charts_tips_and_tricks/#automatically-roll-deployments
+        checksum/configmap: {{ include (print $.Template.BasePath "/common/configmap.yaml") . | sha256sum }}
+        checksum/secrets: {{ include (print $.Template.BasePath "/common/secrets.yaml") . | sha256sum }}
+    spec:
+      containers:
+        - name: app
+          image: "{{ .Values.appImage.repository }}:{{ .Values.appImage.tag }}"
+          command: ["foreman"]
```

Comment:
This is a bit outside of my lane, but I don't think we want to use foreman here, as it's not intended for production use and introduces an extra level of indirection between the container's entrypoint and the actual Rails process. That is primarily an issue for the container/Dockerfile though, which we should likely look at separately from this PR.

Reply:
this is how we ran it in ECS, I'm no ruby expert too: https://github.com/alphagov/govuk-infrastructure/blob/fe361ded9480b71014fe3dc4c6912a04f2a2f370/terraform/deployments/govuk-publishing-platform/app_publisher.tf#L86

Reply:
Yeah that will work, but it's not really a production-grade setup. We can dig into that separately from this PR though.